modules/ossm-recommended-resources.adoc

```diff
@@ -4,11 +4,26 @@ This module included in the following assemblies:
 ////
 
 [id="ossm-recommended-resources_{context}"]
-= Recommended compute resources
+= Setting limits on compute resources
 
-By default, `spec.proxy` has the settings `cpu:10m` and  `memory:128M`.  If you are using Pilot, `spec.runtime.components.pilot` has the same default values.
-The settings in the following example are based on 1,000 services and 1,000 requests per second. You can change the values for `cpu` and `memory` as necessary based on your environment.
+By default, `spec.proxy` has the settings `cpu: 10m` and  `memory: 128M`.  If you are using Pilot, `spec.runtime.components.pilot` has the same default values.
```

Comment:
Is there an extra space between these two sentences?

Reply:
Not any more :)